janitor/helpers/log_helpers.py

```diff
@@ -0,0 +1,39 @@
+import os
+from datetime import datetime
+from pathlib import Path
+from typing import Optional
+
+TMP_FOLDER = Path("./janitor/tmp")
```

Comment:
What is the root path? If we make changes in future affecting it, this could change the TMP_FOLDER location as well. I think it would be better if this is a full path instead of a relative, supported by a config variable to allow changing it.

Reply:
I've moved this path to now be in the config `defaults`, the root path will be the root of this repository (pointing to `run.py` at the top level) so I don't think it matters too much for local machines since `janitor` is not really used in this way, and the path can be specified more explicitly for the deployment.

Reply:
Then, is TMP_FOLDER definition here needed then anymore?